### PR TITLE
Add PostgreSQL to Usage

### PR DESCRIPTION
**Pull request by jpalac**
Implementation of https://github.com/Particular/ServiceControl/issues/4254

### CODE REVIEW

File: src/Frontend/src/views/throughputreport/transport.ts [hunk 1]
```diff
@@ -7,4 +7,5 @@ export enum Transport {
   RabbitMQ = "RabbitMQ",
   SQLServer = "SqlTransport",
   AmazonSQS = "AmazonSQS",
+  PostgreSQL = "PostgreSql",
```
Comment (johnsimons) [inline src/Frontend/src/views/throughputreport/transport.ts:10]: I assume this matches with the key in the report in SC?

**Review by johnsimons — APPROVED** [review]

File: src/Frontend/src/views/throughputreport/transport.ts (hunk 1 above)
Comment (andreasohlund) [inline src/Frontend/src/views/throughputreport/transport.ts:10]: Nitpick: The spelling seems to be "PostgreSQL" (https://www.postgresql.org/) should we align with that?

Comment (jpalac) [reply src/Frontend/src/views/throughputreport/transport.ts:10]: We do - the left hand side is the enum value and the right hand side is whats coming from SC so we are using that just to match.

Comment (andreasohlund) [reply src/Frontend/src/views/throughputreport/transport.ts:10]: Ah, should we adjust the sc side in that case to be consistent?

Comment (jpalac) [reply src/Frontend/src/views/throughputreport/transport.ts:10]: It's internal only - in line with the project name.
I don't mind. Sql transport does something similar where the name of project etc uses Sql, then it shows externally as SQL So in the postgres naming we followed that pattern.
I'm fine either way.

Comment (andreasohlund) [reply src/Frontend/src/views/throughputreport/transport.ts:10]: Ok, fair enough 👍

**Review by PhilBastian — APPROVED** [review]

**Review by SzymonPobiega — APPROVED** [review]

**Review by andreasohlund — APPROVED** [review]

**Pull request merged**